tokio-console/src/state/mod.rs

```diff
@@ -218,16 +229,41 @@ impl State {
 
     pub(crate) fn update_task_details(&mut self, update: proto::tasks::TaskDetails) {
         if let Some(id) = update.task_id {
-            let details = Details {
-                span_id: id.id,
-                poll_times_histogram: update.poll_times_histogram.and_then(|data| {
-                    hdrhistogram::serialization::Deserializer::new()
-                        .deserialize(&mut Cursor::new(&data))
-                        .ok()
-                }),
+            let span_id = id.id;
+            let task_id = self.tasks_state.ids.id_for(span_id);
+            let task_ref = self.tasks_state.task(task_id).unwrap();
+            let task_ref = task_ref.upgrade().unwrap();
```

Comment:
i think if we get an update for a task whose ID we've never seen before, we _probably_ shouldn't panic --- it means the server almost certainly did something wrong, but it would probably be better for the console CLI to print some kind of error and keep going, skipping that update, instead of crashing...